app/resources/admin.html.template

```diff
@@ -1077,10 +1077,10 @@
       </div>
     </div>
     <div class="config">
-      <label for="amp_analytics_id">AMP Analytics ID (optional):</label>
+      <label for="amp_gtm_id">AMP Analytics ID (optional):</label>
```

Comment:
Can you rename the label to "AMP GTM ID" as well?

Reply:
Yes, sorry about that.